fastlane/lib/fastlane/commands_generator/help.erb

```diff
@@ -0,0 +1,35 @@
+  <%= program :name %>
```

Comment:
This file was brought from here.
https://github.com/fastlane/commander/blob/master/lib/commander/help_formatters/terminal_compact/help.erb

I also did some tweaks to substitute these helpers with inline code.
https://github.com/fastlane/commander/blob/master/lib/commander/runner.rb#L200-L223